snippets/standard/buffers/memory-t/void-returning-async/void-returning-async.cs

```diff
@@ -0,0 +1,56 @@
+﻿// <Snippet1>
+using System;
+using System.Buffers;
+using System.IO;
+using System.Threading.Tasks;
+
+public class Example
+{
+    // <Snippet1>
+    // An acceptable implementation.
+    static void Log(ReadOnlyMemory<char> message)
```

Comment:
Same comment on async as the previous samples.

Reply:
That won't work, since spans aren't allowed in async methods.

Reply:
@rpetrusha @BillWagner does the code use [this `ReadOnlyMemory` struct](https://github.com/dotnet/corefx/blob/master/src/Common/src/CoreLib/System/ReadOnlyMemory.cs)? Then it's not `ref struct`, but usual `readonly struct`. Then, I expect it can be used in `async` methods. Or I'm wrong here?

Reply:
Oh, I see: though the `Log` might be `async`, it, anyway, cannot be awaited in the `Main` as `Main` contains spans and cannot be `async`. And not awaiting `async` methods might be not a good example.